src/asmcnc/skavaUI/screen_homing_active.py

```diff
@@ -170,5 +168,6 @@ def update_strings(self):
     def windows_cheat_to_procede(self):
         if sys.platform == "win32" or sys.platform == "darwin":
             self.return_to_ec_if_homing_not_in_progress()
+            self.m.is_machine_homed = True
         else:
             pass
```

Comment:
Remove the else: pass also while here ;)

Reply:
Hahaha